infra/scripts/seed-roles.sh

```
@@ -0,0 +1,92 @@
+
+# Seed Roles Script
+# This script seeds the roles table with initial data
+
+set -e
+
+echo "🌱 Seeding roles data..."
+echo "========================="
+
+# Load environment variables
+if [ -f "docker/.env" ]; then
+    export $(cat docker/.env | grep -v '^#' | xargs)
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)